### DISCUSSION TITLE
SlackAppのインストール希望

### DISCUSSION THREAD

**Discussion by yamahubuki** [nvdajp/nvdajp#322] [アイディア]
アドオン開発者です。
NVDA日本語ベータ版・アルファ版のリリースにいち早く気づき、事前の動作検証等を円滑に進め、正式版リリースに間に合わせる手段を探しております。

GitHub公式のメール通知もありますが、タグの通知機能がないため使いにくく思っています。

そこで、このリポジトリにSlackのアプリをインストールして頂けないでしょうか。
受信側はSlackのワークスペースが必要になってしまいますが、それさえ満たせれば必要な通知の範囲をカスタマイズして受信でき、非常に便利かと思っています。

私のようなアドオン開発者は、タグの通知を受信することでα版・β版・正式版それぞれのリリースがあった際に気づくことができます。

Slackのアプリは通知の機能のみですので、特に問題はないと考えています。
ご検討お願い致します。

**Comment by nishimotz** [comment]
ご提案ありがとうございます。

下記のことを書かれているのだと思います
https://slack.com/intl/ja-jp/help/articles/232289568-GitHub-%E3%81%A8-Slack-%E3%82%92%E9%80%A3%E6%90%BA%E3%81%95%E3%81%9B%E3%82%8B

「受信側はSlackのワークスペースが必要」とのことですが、
GitHub と Slack を連携するときは相互にいろいろ権限を付与することになります。

NVDA 日本語チームが Slack ワークスペースを開設して、
GitHub nvdajp リポジトリの通知を受けたい人は、
NVDA 日本語チームの Slack に参加していただく、
という運用であれば、似たことは他でやっているので、技術的にはできると思います。

提案されていることはそれとは異なるでしょうか？

**Comment by yamahubuki** [comment]
ご確認ありがとうございます。
Slackの権限は相互付与であることを完全に失念しておりました。

やはり、原則に立ち返ってAPIを毎日叩くスクリプトを組みたいと思います。
